_src/_posts/2017-09-11-importance-of-serving-ads.markdown

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: The importance of serving the right amount of ads 
+date:   2017-09-11
+tag: data-science
+categories:
+ - data-science
+keywords: "ROI, impressions, clicks, advertising, data science, impression frequency cap, CPA, quality, view through attribution"
+description: In this post we analyze how different levels of impressions affect the 
+impact of advertising in multiple campaigns.
+author: jdemonasterio
+---
+
+
+<!--excerpt.start-->
+
+[Jampp] [jampp] helps customers boost their mobile sales with performance marketing. One of the most challenging problems of the industry is the impact quantification an ad has on users. A poorly tackled problem is understanding when a user has been affected enough by ads, and why acquiring clicks is not always best aligned with the advertiser's interests.
+We know that, sometimes, serving more impressions to users will result in CPA increase, since we can not influence their behavior any more. The key question is: how many ads are enough? 
+
+<!--excerpt.end→
```

Comment:
fake comment, trying to re-open this PR as per stackoverflow help

https://stackoverflow.com/questions/12674304/github-reopening-a-merged-pull-request